examples_tests/22.RaytracedAO/dirty_source/ExtraCrap.cpp

```diff
@@ -21,7 +21,7 @@ constexpr uint32_t kOptiXPixelSize = sizeof(uint16_t)*3u;
 core::smart_refctd_ptr<ICPUSpecializedShader> specializedShaderFromFile(IAssetManager* assetManager, const char* path)
 {
 	auto bundle = assetManager->getAsset(path, {});
-	return core::move_and_static_cast<ICPUSpecializedShader>(*bundle.getContents().begin());
```

Comment:
is this why the sample crashed at quit?

Reply:
yep, you've moved from a cached bundle

Reply:
yeah, we need to make a way on the `smart_refctd_ptr/refctd_dynamic_array` which is being used in the `SBundle` to forbid that insanity

Reply:
this is what the `inny concern (design bug)` should be about (not moving from GPU conversion result... I got my wires crossed)

Reply:
yea, i figured